shipper.go

```diff
@@ -41,49 +140,29 @@ func startShipper(configFile string, logger log.Logger) {
 		os.Exit(1)
 	}
 
-	ticker := time.NewTicker(5 * time.Second)
-	defer ticker.Stop()
-	for range ticker.C {
-		blockDirs, err := ioutil.ReadDir(".")
-		if err != nil {
-			level.Error(logger).Log("error", err.Error())
-			os.Exit(1)
-		}
-		blocks := make([]string, 0, len(blockDirs))
-		for _, bd := range blockDirs {
-			if bd.Name() == "wal" {
-				continue
-			}
+	dirName, err := ioutil.TempDir("", "tempData")
```

Comment:
Can we call it `tmpDir` instead of `dirName`? Makes it hard to interpret the functions below.